src/thunderbots/software/ai/hl/stp/tactic/free_kick_tactic.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "ai/hl/stp/tactic/tactic.h"
+
+/**
+ * This tactic is used when performing a free kick, and no pass is available.
+ * It is also assumed that the ball is stationary, in preparation for a set piece.
+ * 
+ * For now, it attempts to find a shot towards the enemy goal;
+ * if none are found, it will shoot at an enemy robot to get a deflection. 
+ */
+
+class FreeKickTactic : public Tactic
+{
+    public:
+
+        /**
+         * Constructs new FreeKickTactic from World
+         */
+        explicit FreeKickTactic(const World& world, bool loop_forever); 
```

Comment:
I'd recommend not passing in the entire world if we don't need it. Same goes for the update function

Reply:
Seems like the chip and chase eval only uses `World`. Since this tactic uses everything in `World` other than gamestate I'm in favour of keeping its usage to make things simpler.